tests/cases/elements/toucharea.slint

```diff
@@ -50,10 +50,10 @@ TestCase := Rectangle {
                 pointer-event-test += "left";
             } else if (e.button == PointerEventButton.middle) {
                 pointer-event-test += "middle";
-            } else if (e.button == PointerEventButton.none) {
-                pointer-event-test += "none";
-            } else {
+            } else if (e.button == PointerEventButton.other) {
```

Comment:
Is this something for the syntax updater perhaps?

Reply:
Good point